README.md

```diff
@@ -67,6 +67,25 @@ export const ExampleComponent = () => {
 
 </details>
 
+## Installation
+
+<details>
```

Comment:
Maybe I have not it stated it good enough in the ticket. I wanted to have two `details` section. One for yarn and one for npm.

Should look like [this gist](https://gist.github.com/igeligel/7fb12f8808c15f777d66e03df6a074cd).